website/siteConfig.js

```diff
@@ -257,15 +279,17 @@ const siteConfig = {
   projectName: 'jest',
   repo: 'facebook/jest',
   users,
+  videos,
+  videoTypes,
   editUrl: repoUrl + '/edit/master/docs/',
   headerLinks: [
-    {doc: 'getting-started', label: 'Docs'},
-    {doc: 'api', label: 'API'},
-    {page: 'help', label: 'Help'},
-    {blog: true, label: 'Blog'},
-    {languages: true},
-    {search: true},
-    {href: repoUrl, label: 'GitHub'},
+    { doc: 'getting-started', label: 'Docs' },
```

Comment:
Can you revert these whitespace changes? Please run `yarn lint`!

Reply:
I think there's no `lint` script. 🤷🏽‍♂️